app/services/git_wrapper.rb

```diff
@@ -36,4 +45,18 @@ def remove_dir
     FileUtils.rm_rf(@repo_dir)
     ReturnValue.ok
   end
+
+  private
+
+  def generate_procfile(web_processes)
+    file_text = web_processes.to_a.map { |name_command_pair| name_command_pair.join(": ") }.join("\n")
+    procfile = File.new(File.join(@repo_dir, PROCFILE_NAME), "w")
```

Comment:
I think that git wrapper is not an appropriate place for generating procfile. It probably ok for now, but later we should extract it

Reply:
Agree, for now we do all the stuff with the cloned repo.